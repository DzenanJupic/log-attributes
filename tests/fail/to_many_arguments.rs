#[log_attributes::log(info, "{fn} was executed", asdf)]
fn to_many_arguments() {}

fn main() {}
