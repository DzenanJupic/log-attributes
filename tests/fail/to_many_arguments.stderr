error: Unexpected argument
 --> tests/fail/to_many_arguments.rs:1:50
  |
1 | #[log_attributes::log(info, "{fn} was executed", asdf)]
  |                                                  ^^^^
